.github/workflows/docker-build-push.yml

```diff
@@ -120,6 +120,20 @@ jobs:
           docker tag ${{ steps.check_docker_images.outputs.runtime_tag }} $TO_TAG
           docker push $TO_TAG
 
+      - name: Deploy main-nightly image
```

Comment:
Shouldn't this nightly version have its own workflow yml file? a scheduled one?

Reply:
We could put it in a separate file, I figured that we would want to push it on the same schedule as the build. Easy either way.